### PR TITLE
fix: updated Figma links

### PR DESCRIPTION
**Pull request by zetareticoli**
Fixed prototype links.

### CODE REVIEW

**Review by Fupete — APPROVED** [review]

**Comment by zetareticoli** [comment]
@Fupete There's still a wrong link, wait for the fix before proceding with merge

**Comment by Fupete** [comment]
@zetareticoli switched to draft. Just mark this "Ready for review" when done.

**Pull request merged**
